flink-end-to-end-tests/pom.xml

```diff
@@ -153,6 +170,32 @@ under the License.
 
 	<build>
 		<plugins>
+			<plugin>
+				<artifactId>maven-resources-plugin</artifactId>
+				<!-- <version>3.1.0</version> -->
+				<executions>
+					<execution>
+						<id>copy-resources</id>
+						<!-- here the phase you need -->
```

Comment:
```suggestion
```

Reply:
Sorry that I didn't catch this in my self-review